### ISSUE TITLE
[Bug] Crash when placing the Starter Block

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#4821]
### Short Description of the issue:
I placed a Starter Block and sevo froze and became unresponsive, I had to force close the game

### Steps to reproduce the issue:
- Place a Starter Block

### Environment
- Starship EVO version: 22w31a
- Graphics Card: RX 580

### Additional information:
[Player.log](https://github.com/tsunamayo/Starship-EVO/files/9277054/Player.log)
Another of the same experience - [Player-prev.log](https://github.com/tsunamayo/Starship-EVO/files/9277062/Player-prev.log)

**Comment by Uncle-Ulty** [comment]
same here...

**Comment by ProPeach** [comment]
I should note - the crash didn't occur *every time*, I was able to spawn in a car from the Workshop with the Starter Block in the same session, then a minute later I placed another Starter Block to spawn one of my own bps and the crash occurred. I loaded up the game and the same world again, placed a Starter Block and got the same crash so it seems repeatable after you get one

**Comment by tsunamayo** [comment]
@ProPeach Hi, on what world were you? Voxel 6? I can spawn without any issues here...
Unfortunately there is nothing in the log, do you have some crash dump somewhere maybe? 
They should be over there:
C:/Users/[YOUR USER]/AppData/Local/Temp/Moonfire Entertainment/Starship EVO/Crashes
Thanks

**Comment by ProPeach** [comment]
Thanks for checking in, this was the world -
[Performance Test.zip](https://github.com/tsunamayo/Starship-EVO/files/9277707/Performance.Test.zip)
 
I don't have any crash reports from that session sadly, perhaps because the game became unresponsive and I had to manually close it, rather than it crashing outright?

**Comment by tsunamayo** [comment]
ok I am reproducing on that world! God the game is buggy af lately... I keep you posted!

**Comment by Kaiser-Indrasil** [comment]
> ok I am reproducing on that world! God the game is buggy af lately... I keep you posted!

@tsunamayo Perhaps it's a good idea to make more use of the Experimental branch. We need a stable Main and we could troubleshoot at Experimental before merging the new features on Main. I think 2 week delay for each feature patch that isn't a hotfix would be sufficient.

**Comment by tsunamayo** [comment]
@ProPeach Okay it is finally fixed - I am pushing a build soon! I am paying some over-engineering from a long time ago.

@Kaiser-Indrasil Yeah, the problem is I had to push to default to fix some crash and the networking issue.
For that particular issue I am not sure if it is was linked to recent build, I would say it might have been > 1 month old (related to World Collider, and I did not touched that recently), so maybe there was not a lot of people on the experimental or maybe it was a hard to get issue, not sure.

Cheers
